instrumentation/go.mongodb.org/mongo-driver/mongo/otelmongo/config.go

```diff
@@ -57,11 +55,3 @@ func WithTracerProvider(provider trace.TracerProvider) Option {
 		cfg.TracerProvider = provider
 	}
 }
-
-// WithCommandAttributeDisabled specifies if the MongoDB command is added as an attribute to Spans or not.
-// The MongoDB command will be added as an attribute to Spans by default if this option is not provided.
-func WithCommandAttributeDisabled(disabled bool) Option {
```

Comment:
Is there a reason this is being removed? Didn't this resolve https://github.com/open-telemetry/opentelemetry-go-contrib/issues/712?

Reply:
This was being used only to determine whether to include the "full" command in the `db.statement` attribute.

But according to the [example MongoDB attributes](https://github.com/open-telemetry/opentelemetry-specification/blob/2a712f7626cb62b21ae210ef3175612e7744f285/specification/trace/semantic_conventions/database.md#mongodb) given in the DB semantics tags specification
   * `db.statement` is *unset*
   * `db.operation` is set to the name of the operation
There doesn't appear to be an appropriate attribute for the full "command".

Let me survey other implementations.

Reply:
Looks like other implementations do set `db.statement`. Best implementation seems to be the [Java autoinstrumentation ](https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/0b0516bd1a0599c8f536cc6b2151782000681f39/instrumentation/mongo/mongo-3.1/library/src/main/java/io/opentelemetry/instrumentation/mongo/v3_1/MongoClientTracer.java)where it both sanitizes the command and constrains it to a maximum size.

I've restored setting `db.statement` and the `WithCommandAttributeDisabled()`, but with a TODO for better sanitizing and max size limit.

Also reorganized the tests a little including adding a check for `SpanKindClient`